### PR TITLE
Make test.sh executable using chmod

### PR DESCRIPTION
**Pull request by JosephCarpenter**

### CODE REVIEW

File: robot/tests/.pytest_cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ArchdukeTim) [inline robot/tests/.pytest_cache/v/cache/lastfailed:1]: don't use -a if you don't know what's being committed

**Review by ArchdukeTim — CHANGES_REQUESTED** [review]
You have files that seem to have gotten moved in when you ran `git commit -a`

**Comment by JosephCarpenter** [comment]
Huh. I just used -a because I fixed a couple instances of extraneous white space when I ran test.sh, and didn't think it was worth it's own commit. I don't know why any files were moved.

File: robot/tests/.pytest_cache/v/cache/lastfailed (hunk 1 above)
Comment (JosephCarpenter) [reply robot/tests/.pytest_cache/v/cache/lastfailed:1]: I thought I did know what I had committed, but I didn't see anything about that. What is it?

Comment (ErikBoesen) [reply robot/tests/.pytest_cache/v/cache/lastfailed:1]: It's just some junk files that `pytest` uses. I just opened #27; add it to the `.gitignore` file and redo this commit.

**Comment by JosephCarpenter** [comment]
I have amended it.

**Comment by ErikBoesen** [comment]
You still have the `.pytest_cache` directory.

**Comment by ErikBoesen** [comment]
Didn't notice you had this already. You may want to redo the commit.

**Comment by ArchdukeTim** [comment]
You also have changes that have nothing to do with test.sh 😂

**Comment by ErikBoesen** [comment]
Maybe do the PR again

**Pull request closed without merging**